lib/compat.js

```diff
@@ -15,276 +15,7 @@
  * along with Adblock Plus.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-//
```

Comment:
The ABP copyright above this could probably be removed too. Does this commit need to be in the firefox compat pull request? It seems kind of orthogonal.